janusgraph-test/src/test/java/org/janusgraph/graphdb/GraphDatabaseConfigurationInstanceIdTest.java

```diff
@@ -51,35 +45,34 @@ public void graphShouldOpenWithSameInstanceId() {
         final StandardJanusGraph graph1 = new StandardJanusGraph(new GraphDatabaseConfigurationBuilder().build(new CommonsConfiguration(config)));
 
         assertEquals(graph1.openManagement().getOpenInstances().size(), 1);
-        assertEquals(graph1.openManagement().getOpenInstances().toArray()[0], "not-unique");
+        assertEquals(graph1.openManagement().getOpenInstances().toArray()[0], "not-unique(current)");
```

Comment:
I think you accidentally added to `non-unique` strings in a lot of places the suffix `(current)`. Please, check [this amend](https://github.com/JanusGraph/janusgraph/compare/a9d4ad380c399f734d2e8d026f44daf1d4f51eb0..f99c14f5fe55d1b673242ca474ba0ab6d2387f9c) and remove `(current)` suffix from all those strings.

Reply:
I didn't work without suffix `(current)`.

Reply:
Please change the line:
`assertEquals(graph1.openManagement().getOpenInstances().toArray()[0], "not-unique(current)");`
to line:
`assertEquals(graph1.openManagement().getOpenInstances().toArray()[0], "not-unique");`